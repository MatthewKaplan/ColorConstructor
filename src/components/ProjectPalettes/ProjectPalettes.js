import React, { Component } from "react";
import PieGraph from "../PieGraph/PieGraph";
import DoughnutGraph from "../DoughnutGraph/DoughnutGraph";

class ProjectPalettes extends Component {
  render() {
    return (
      <div className="project-palettes-component">
        <div className="graphs">
          <section className="pie-graph">
            <PieGraph colors={this.props.colors} />
          </section>
          <section className="pie-graph">
            <DoughnutGraph colors={this.props.colors} />
          </section>
        </div>
      </div>
    );
  }
}

export default ProjectPalettes;
