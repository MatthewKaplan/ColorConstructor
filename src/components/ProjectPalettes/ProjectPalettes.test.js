import React from "react";
import { shallow } from "enzyme";
import ProjectPalettes from "./ProjectPalettes";

describe("ProjectPalettes", () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallow(<ProjectPalettes />);
  });

  it("should match the snapshot", () => {
    expect(wrapper).toMatchSnapshot();
  });
});
