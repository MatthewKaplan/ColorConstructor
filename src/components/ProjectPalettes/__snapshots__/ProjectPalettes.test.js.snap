// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ProjectPalettes should match the snapshot 1`] = `ShallowWrapper {}`;
